app/Console/Kernel.php

```diff
@@ -26,9 +26,9 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        $schedule->job(new PickupBookJob())->dailyAt('12:00');
+        $schedule->job(new PickupBookJob())->hourly();
 
-        $schedule->job(new BookReturnReminderJob())->dailyAt('12:00');
+        $schedule->job(new BookReturnReminderJob())->dailyAt('00:00');
```

Comment:
I did not mind for this to be at 12.00. I really personally dislike when I receive email in the night! :)

Reply:
The reason I made it hourly was simply cos of the request being open for you 24 hours... So it has to be run every hour to capture that but I think I'll just create a separate cron job for that then.

Reply:
@Zelig880 
The reason I made it hourly was simply cos of the request being open for you 24 hours... So it has to be run every hour to capture that but I think I'll just create a separate cron job for that then.

Reply:
@Zelig880 
Ohh 
Sorry, I was talking about something else.